src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -163,13 +181,27 @@ export class CourseManagementRequests {
             method: 'DELETE',
         });
     }
+
+    /**
+     * Because the only difference between course exercises and exam exercises is the "course" or "exerciseGroup" field
+     * This function takes an exercise template and adds one of the fields to it
+     * @param exercise the exercise template
+     * @param body the exercise group or course the exercise will be added to
+     * */
+    private static getCourseOrExamExercise(exercise: object, body: { course: any } | { exerciseGroup: any }) {
+        const isExamExercise = body.hasOwnProperty('exerciseGroup');
+        const exerciseCopy: any = { ...exercise };
+        const fieldName = isExamExercise ? 'exerciseGroup' : 'course';
+        exerciseCopy[fieldName] = body[fieldName];
+        return exerciseCopy;
+    }
```

Comment:
I think we can write this method even shorter with the current `body` parameter:
```suggestion
    private static getCourseOrExamExercise(exercise: object, body: { course: any } | { exerciseGroup: any }) {
        return Object.assign({}, exercise, body);
    }
```